test/web-platform-tests/to-upstream/html/semantics/forms/the-label-element/clicking-noninteractive-content.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Label event handling when a descendant labelable but not interactive element is clicked</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<label id=label></label>
+<template id=labelable-not-interactive-content>
+  <meter></meter>
+  <output></output>
+  <progress></progress>
+</template>
+
+<script>
+"use strict";
+
+const labelableNotInteractiveElements =
+  Array.from(document.getElementById("labelable-not-interactive-content").content.children);
+const label = document.getElementById("label");
+
+// This part is may be subject to platform-dependent operations in the spec, so we only check for obvious errors.
+// (Clicking once should register at least one click, but less than 30 clicks.)
```

Comment:
I would be curious on the cross-browser results if we changed this to check for exactly one click. (Something to consider for the above-mentioned HTML spec issue, probably.)